src/api/app/models/workflow/step/set_flags.rb

```diff
@@ -6,7 +6,7 @@ class Workflow::Step::SetFlags < Workflow::Step
   validate :validate_flags
 
   def call
-    return if workflow_run.closed_merged_pull_request? || workflow_run.reopened_pull_request?
+    return if workflow_run.closed_merged_pull_request? || workflow_run.reopened_pull_request? || workflow_run.unlabeled_pull_request?
```

Comment:
Why skip this step for labeling?

Reply:
Same as above, there is nothing more to do in this step for `unlabeled` action. For `labeled` action it will execute.